detection_rules/etc/version.lock.json

```diff
@@ -9725,25 +9739,25 @@
       }
     },
     "rule_name": "AWS CloudWatch Log Stream Deletion",
-    "sha256": "e93839d1fa0ad3b30a87027cfc1a0c51d8008c86ae5891f0ca2bf8cdcce1172d",
+    "sha256": "5938c4ca763d90161f10cc3da2d26dfbd70bd5e4f2fb947388e17fce6692498d",
     "type": "query",
-    "version": 103
+    "version": 104
   },
   "d62b64a8-a7c9-43e5-aee3-15a725a794e7": {
     "min_stack_version": "8.3",
     "previous": {
       "7.16": {
         "max_allowable_version": 99,
         "rule_name": "GCP Pub/Sub Subscription Creation",
-        "sha256": "29519fb575a745714635c63fc1316d44e4760dc28d15c7bb88a5665b423c5330",
+        "sha256": "5d074c906776ecd8e5847fb793728b81b80895a83cf706d49341241756921dbc",
```

Comment:
Forked rule changed when all rules are min-stacked at 8.3 and thus changes would not backport to this branch.